framework/source/class/qx/ui/form/VirtualSelectBox.js

```diff
@@ -591,6 +599,15 @@ qx.Class.define("qx.ui.form.VirtualSelectBox",
       this.__filterUpdateRunning--;
     },
 
+    _applyDelegate : function(value, old) {
+      if (old && old.configureItem && !value.configureItem) {
+        value.configureItem = function(item) {
+          item.setRich(true);
+        };
+      }
```

Comment:
this should only happen if incremental search is active. Is it actually necessary? does setDelegate override existing properties ?

Reply:
I think this is not neccesary in this way, as delegates get applied when they are configured ... configuring a new/different delegate does not override the existing ones ... the code for the handling of delegates is in the controller, which seems a bit odd as this means that `setDelegate` could act differently depending on where it is implemented ... but at least in the listController which I think is in us here, multiple invocations with different properties do not seem to interact ... 

https://github.com/qooxdoo/qooxdoo/blob/9e375001ea22a9f4a1ea79c9df353c8f9d5df177/framework/source/class/qx/data/controller/List.js#L281

Reply:
I think `delegate.configureItem` must handle `rich` explicitly when changing `delegate.filter` as items are dynamically generated.